nvm.psm1

```diff
@@ -263,6 +263,13 @@ function Install-NodeVersion {
             }
             else {
                 $Architecture = [System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture
+
+                if (Get-Command -Name uname -ErrorAction SilentlyContinue) {
```

Comment:
What's the value of `$Architecture` on a Pi?

Reply:
On a Pi 4, it's "Arm"